docs/modules/robots.rst

```diff
@@ -35,130 +35,174 @@ Models
-
-LBR IIWA 7
-~~~~~~~~~~
-.. image:: ../images/models/robot_model_IIWA.png
-   :alt: iiwa_robot
-   :align: left
-   :width: 50%
-
-- DoF: 7
-- ...
-- ...
-
-Jaco
-~~~~
-.. image:: ../images/models/robot_model_Jaco.png
-   :alt: jaco_robot
-   :align: left
-   :width: 50%
-
-- DoF: 7
-- ...
-- ...
-
-Kinova Gen3
-~~~~~~~~~~~
-.. image:: ../images/models/robot_model_Kinova3.png
-   :alt: kinova3_robot
-   :align: left
-   :width: 50%
-
-
-
-- DoF: 7
-- Parallel jaw gripper
-- Fixed base
-
-UR5e
-~~~~
-.. image:: ../images/models/robot_model_UR5e.png
-   :alt: ur5e_robot
-   :align: left
-   :width: 50%
-
-- DoF: 6
-- Parallel jaw gripper
-- Fixed base
-
-Baxter
-~~~~~~
-.. image:: ../images/models/robot_model_Baxter.png
-   :alt: baxter_robot
-   :align: left
-   :width: 50%
-
-- DoF: 14
-- ...
-- ...
-
-Tiago
-~~~~~~
-.. image:: ...
-   :alt: tiago_robot
-   :align: left
-   :width: 50%
-
-- DoF: ...
-- ...
-- ...
-
-GR1
-~~~~~~
-.. image:: ...
-   :alt: gr1_robot
-   :align: left
-   :width: 50%
-
-- DoF: ...
-- ...
-- ...
+
+.. list-table::
+   :widths: 15 50 35
+   :header-rows: 1
+
+   * - Robot
+     - Image
+     - Description
+   * - **Panda**
+     - .. image:: ../images/models/robot_model_Panda_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 7
+       - **Default Gripper:** PandaGripper
+       - **Default Base:** RethinkMount
+   * - **Sawyer**
+     - .. image:: ../images/models/robot_model_Sawyer_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 7
+       - **Default Gripper:** RethinkGripper
+       - **Default Base:** RethinkMount
+   * - **IIWA**
+     - .. image:: ../images/models/robot_model_IIWA_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 7
+       - **Default Gripper:** Robotiq140Gripper
+       - **Default Base:** RethinkMount
+   * - **Jaco**
+     - .. image:: ../images/models/robot_model_Jaco_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 7
+       - **Default Gripper:** JacoThreeFingerGripper
+       - **Default Base:** RethinkMount
+   * - **Kinova3**
+     - .. image:: ../images/models/robot_model_Kinova3_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 7
+       - **Default Gripper:** Robotiq85Gripper
+       - **Default Base:** RethinkMount
+   * - **UR5e**
+     - .. image:: ../images/models/robot_model_UR5e_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 6
+       - **Default Gripper:** Robotiq85Gripper
+       - **Default Base:** RethinkMount
+   * - **Baxter**
+     - .. image:: ../images/models/robot_model_Baxter_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 14
+       - **Default Gripper:** RethinkGripper
+       - **Default Base:** RethinkMount
+   * - **GR1**
+     - .. image:: ../images/models/robot_model_GR1_isaac.png
+          :width: 90%
+          :align: center
+     - - **DoF:** 14
```

Comment:
I think the dof is 6 + 6 (arms) + 6 + 6 (hands) + base/torso/legs(?)

Reply:
Fixed